File: HD/hw22/CmdArgs.java
/* Example of command-line arguments in Java 
   Prints any command-line arguments, one per line
   Usage:  javac CmdArgs.java
           java CmdArgs arg1 arg2 ...  */

import java.io.*;

public class CmdArgs {

  public static void main(String[] args) {
    for (int i = 0;  i < args.length;  i++)
      System.out.println(args[i]);
  }
}

File: HD/hw22/cmdargs.c
/* Example of command-line arguments in c
   Prints any command-line arguments, one per line
   Usage:  gcc -o cmdargs2 cmdargs.c
           cmdargs2 arg1 arg2 ...  */

#include <stdio.h>

int main(int argc, char **argv) {
  int i;  /* loop control */
  for (i = 0;  i < argc;  i++)
    printf("%s\n", argv[i]);
  
  return 0;
}
